modules/nextflow/src/main/groovy/nextflow/data/config/DataConfig.groovy

```diff
@@ -31,8 +31,11 @@ class DataConfig {
 
     final DataStoreOpts store
 
+    final boolean enabled
+
     DataConfig(Map opts) {
         this.store = new DataStoreOpts(opts.store as Map ?: Map.of())
+        this.enabled = opts.store != null
```

Comment:
We may decided to an explicit logic to enable/disable it. Just a temporay flag

Reply:
I have added the workflow.data.enabled flag to enable/disable it